### ISSUE TITLE
In New Tab You Have To Click Reload

### ISSUE THREAD

**Issue by erievs** [erievs/FourthTube#7]
Kind of annoying

**Comment by SeaborneProto** [comment]
is this really an *issue* though? the OG thirdtube client had it like this aswell

**Comment by erievs** [comment]
> is this really an _issue_ though? the OG thirdtube client had it like this aswell
> 
> [this is just imo, though]

I don't care if it is ThirdTube, it's an issue.

**Comment by swelted** [comment]
> > is this really an _issue_ though? the OG thirdtube client had it like this aswell
> > [this is just imo, though]
> 
> I don't care if it is in ThirdTube, it's an issue.

Yeah. If you decide to take up the burden of a project, officially or not, the problems the dev of the old project left are still your problem

**Comment by Smu1zel** [comment]
Fixed in Beta 29.
